packages/components/src/styles/table-cell/table-cell.stories.ts

```diff
@@ -60,6 +61,50 @@ export const Samples = {
         return args['default-slot'] ?? 'Lorem ipsum dolor sit amet.';
       })
     );
+    const sortData: ('ascending' | 'descending' | 'none')[] = Array.from({ length: tableColumnCount }, (_v, i) =>
+      i === 0 ? 'ascending' : 'none'
+    );
+
+    // Function to exchange the sort icons and set the aria-sort attribute in the sortable table after a click on a table header
+    const sortTable = (column: number) => {
```

Comment:
Could you implement real sorting behaviour instead of just having `Lorem Ipsums` or what do you think?

Reply:
Good point

Reply:
I would then also remove the default slot input in the controls so that it cannot be overwritten in the sample.

Reply:
That was also my comment .. +1

Reply:
current solution fits for me.